soccer/gameplay/plays/sequences/move_sequence.py

```diff
@@ -0,0 +1,22 @@
+import behavior_sequence
+import skills.move
+
+class MoveSequence(behavior_sequence.BehaviorSequence):
+    def __init__(self, positions = None):
+        super().__init__()
+        self._positions = [] if positions == None else positions
+
+    def on_enter_start(self):
+        super().on_enter_start()
+        for pos in self.positions:
+            self.behaviors.append(skills.move.Move(pos))
+
+    @property
```

Comment:
Not sure, but I think this will solve your problem. Can you change line 7 above to use `self.positions` instead of `self._positions`? Let me know if that dosen't work.